src/q7/ml-ELEC2870/exam/2019/Janvier/All/ml-ELEC2870-exam-2019-Janvier-All.tex

```diff
@@ -0,0 +1,63 @@
+\documentclass[en]{../../../../../../eplexam}
+
+\hypertitle{Machine Learning}{7}{ELEC}{2870}{2019}{Janvier}{All}
+{ELEC students 2018-2019}
+{Michel Verleysen and John Lee}
+
+\section{Verleysen}
+
+\begin{itemize}
+    \item \textbf{Generalization (5)}: Explain how overfitting, generalization and complexity are linked together. Illustrate with some examples (in 2 or 3D) for both regression and classification.
+    \item \textbf{Feature selection}: Explain what feature extraction is. Why is it important to do feature extraction to obtain less but not too much features?
+    \item \textbf{Feature selection}: Greedy's algorithm, explain the advantages and drawbacks of the forward and backward, then speak about the compromises between them. Why does Greedy's algorithm belong to the non optimal algorithms?
```

Comment:
Greedy **'s** ? I couldn't find anyone on the Internet talking about Greedy's algorithm.

Reply:
Je vais me renseigner, je ne suis pas le cours :smile: .

Reply:
```suggestion
    \item \textbf{Feature selection}: greedy algorithm, explain the advantages and drawbacks of the forward and backward, then speak about the compromises between them. Why does greedy algorithm belong to the non optimal algorithms?
```